pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -344,7 +344,7 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 	}
 
 	// Generate the objects
-	objs, progressing, err := network.Render(&operConfig.Spec, bootstrapResult, ManifestPath, r.client, r.featureGates)
+	objs, progressing, err := network.Render(&newOperConfig.Spec, bootstrapResult, ManifestPath, r.client, r.featureGates)
```

Comment:
yeah I looked into it, it never really mattered before because Alex updated the bootstrapResult and relied on that to sync the new change rather than the operConfig. But I need to use it now, and I dont think it should hurt anything.

Reply:
This change caused https://issues.redhat.com/browse/OCPBUGS-18517. And yeah, it might not be clever to inject values in Render phase. Please take a look at #1988 and the approach tried there.